conduct.md

```diff
@@ -52,7 +52,7 @@ TODO: Write an introductory paragraph.
 
 ### Participation
 
-Students should participate in a meaningful capacity during this course. This section will outline the acceptable and unacceptable conduct when it comes to class participation. Students should follow these guidelines when participating in classroom activities.
+Students are encouraged to participate in a meaningful capacity during class activities in this course. This section will outline the acceptable and unacceptable conduct when it comes to class participation. Students should follow these guidelines when participating in classroom activities.
```

Comment:
It would be beneficial to add more about what it means to participate in a meaningful capacity: ergo, participating in class discussions, asking questions about the material, etc.